dashboard/app/models/census/state_cs_offering.rb

```diff
@@ -186,7 +203,7 @@ def self.seed_from_csv(state_code, school_year, filename)
         courses = get_courses(state_code, row_hash)
         # state_school_id is unique so there should be at most one school
         school = School.where(state_school_id: state_school_id).first
-        if school
+        if school && state_school_id
```

Comment:
And this extra check doesn't impact any other state imports?

Reply:
Correct.  `state_school_id` is not nullable in state_cs_offering so this is already required.  This extra check is needed because Indiana does not explicitly provide the information needed to construct a state_school_id in their spreadsheet, but they do provide NCES ID, which we can use to lookup `state_school_id` in our `school` model.  Checking that `state_school_id` is not nil handles the case where our lookup for the school by NCES ID fails so we can silently skip it and continue importing the other school courses.